vpr/src/route/route_tree_type.h

```diff
@@ -27,6 +27,13 @@ struct t_linked_rt_edge {
  * parent_switch:  Index of the switch type driving this node (by its        *
  *                 parent).                                                  *
  * inode:  index (ID) of the rr_node that corresponds to this rt_node.       *
+ * ipin:  Net pin index associated with the rt_node. This value ranges from  *
```

Comment:
net_pin_index would be a better name than ipin, both here and in t_trace, in retrospect.  The comment is good; you could change the member name too (here and in t_trace) if you have the energy for it :).

Reply:
Agreed and done.